### PR TITLE
Localized help for `get-help nuget` in Package Manger Console

### LINKED ISSUE #4604: [Test Failure][Localization] The information from command ‘get-help NuGet’ in PMC is not localized
**NuGet Version:** VS.Redist.NuGet VS15.4.0.170213.1779
**VS Version:** D15Rel 26213.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select CHS through ‘Tools -> Options -> International Settings’ and restart VS.
2.  Create a new C# Console Application via VS.
3.  Open the Package Manager Console window, type **‘get-help NuGet’** and hit enter.

**Expected Result:**
Descriptive strings from the command ‘get-help NuGet’ are localized correctly.

**Actual Result:**
Descriptive strings from the command ‘get-help NuGet’ are not localized as below screenshot.
gethelpnuget

**Note:**
It repro for all languages.

### PR DESCRIPTION
**Pull request by dominoFire**
## Bug

Fixes: https://github.com/NuGet/Home/issues/4604
Regression: Yes
* Last working version:   VS 2015
* How are we preventing it in future: Enable localization for help files. See https://github.com/NuGet/Client.Engineering/issues/612

## Fix

- Includes pre-localized files to final, localized VSIX
- Renames help file to keep consistency

## Testing/Validation

Tests Added: No
Reason for not adding tests:  Testing would require locale-sensitive test
Validation:  Manual test:

Example in Turkish:
image

Example in Spanish:
image

Example in English:
image

### CODE REVIEW

**Comment by nkolev92** [comment]
Given that the test is manual...can we add a vendor test? 

*edit*

My bad, meant to click comment :D

File: src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj [hunk 1]
```diff
@@ -179,12 +181,35 @@
...
+      <Lango Include="DEU" Locale="de" />
+      <Lango Include="ESN" Locale="es" />
+      <Lango Include="FRA" Locale="fr" />
+      <Lango Include="ITA" Locale="it" />
+      <Lango Include="JPN" Locale="ja" />
+      <Lango Include="KOR" Locale="ko" />
+      <Lango Include="PLK" Locale="pl" />
+      <Lango Include="PTB" Locale="pt-BR" />
+      <Lango Include="RUS" Locale="ru" />
+      <Lango Include="TRK" Locale="tr" />
```
Comment (donnie-msft) [inline src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj]: Is this an all-inclusive list of the languages we support? Did you copy it from somewhere? Maybe a comment explaining where you got this set.

**Review by donnie-msft — APPROVED** [review]

File: src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj (hunk 1 above)
Comment (dominoFire) [reply src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj]: That's the list of VS supported languages we support. The list is defined at Microbuild.Plugins.Localization. Will add a comment.

Comment (dominoFire) [reply src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj]: I tried to reference internally but, that would require to add imports:

```xml
  <Import Project="$(MicroBuildDirectory)MicroBuild.Core.props"/>
...
  <Import Project="$(MicroBuildDirectory)MicroBuild.Core.targets"/>
```

Comment (donnie-msft) [reply src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj]: I'm happy with just a comment somewhere :)

**Comment by dominoFire** [comment]
Vendors have validated these changes. See internal email

**Pull request merged**